ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/disconnected/private/cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private-workflow.yaml

```diff
@@ -5,6 +5,9 @@ workflow:
     - chain: cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private-provision
     post:
     - chain: cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private-deprovision
+    env:
+      BASE_DOMAIN: "ibmcloud.qe.devcluster.openshift.com"
+      IBMCLOUD_DNS_INSTANCE_NAME: "preserved-openshift-qe-private"
```

Comment:
these value is already set in provision chain, do not need to set them here again.

Reply:
I fail to get the value in ci-operator/step-registry/ibmcloud/deprovision/vpc/ibmcloud-deprovision-vpc-commands.sh, so added it in the workflow

Reply:
ok